src/datalayer/reducers/qr.reducer.js

```diff
@@ -0,0 +1,20 @@
+import { qrAction } from 'constants/actions';
+
+// This state is for getting check-in QR result
+export const INITIAL_STATE = {
+  checkedInResult: {},
```

Comment:
`checkedInResult` or `list`?

Reply:
***checkedInResult*** of course
***list*** is for the list of events get by userId

Reply:
I can not see any action that change `checkedInResult`. And you haven't set initial value for `list`.

Reply:
We will use it when we display the information of ticket. I just leave the success flow as ***Thành công*** not implemented ***checkedInResult*** yet.

Reply:
Oh yeah I forgot the ***list***. It has to be ***checkedInResult***